tools/readme_linter/README.md

```diff
@@ -0,0 +1,22 @@
+# README.md linter
+
+Run readme_linter with the filenames of the readme files you want to lint. For
+example:
+
+```shell
+~/go/src/github.com/influxdata/telegraf$ tools/readme_linter/readme_linter plugins/inputs/file/README.md
```

Comment:
The long path to where you keep telegraf was a bit confusing, how about making this how to build and run it with since if the user is reading this readme, they are probably already here.

```shell
$ go build
$ ./readme_linter <path to README>
```

Reply:
I removed the long paths and added build instructions